activity/activity.info

```diff
@@ -1,6 +1,8 @@
 [Activity]
 name = Adivinanzas 
 bundle_id = org.somosazucar.Adivinanzas
-exec = sugar-activity AdivinanzasActivity.AdivinanzasActivity
+exec = sugar-activity3 AdivinanzasActivity.AdivinanzasActivity
 icon = activity-generic
 activity_version = 3
+repository = https://github.com/sugarlabs/adivinanzas-activity.git
+license = GPLv3+
```

Comment:
Why did you put _license_ here?

Reply:
Added license because of the warning displayed in logs while running the activity.

I am removing it now, there may be some copyright issues of the original author here, I remember you said about copyright and license in a pull request in some other activity previously.

If there is nothing like that, I will commit that again later.